lib/proxy.js

```diff
@@ -29,7 +28,8 @@ export function proxyStringToObject (proxyString) {
     username: parsedUrl.username,
     password: parsedUrl.password
   }
-  const port = toInteger(portString)
```

Comment:
I have seen some comments that the[ lodash implementation ](https://github.com/lodash/lodash/blob/2f79053d7bc7c9c9561a30dda202b3dcd2b72b90/toInteger.js)of toInteger covers more edge cases than built-in parseInt. Do you have any experience with that?
However, in this specific case we should usually not see any weird numbers for the port. So in case there is nothing specific here this would be fine for me. Just wanted to ask if there is anything known.

Reply:
Under the hood, `toInteger` wraps `toNumber` with some extra functionality. There is quite a good SO article explaining the differences between both: https://stackoverflow.com/a/56555471/16181962. 

Long story short since URL already does some pre-processing we dont need to handle all of lodash's edge cases for `toInteger` ([URL.prototype.port](https://nodejs.org/api/url.html#urlport) docs)